packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -17,6 +17,32 @@ import {
   Tip
 } from '@components/shared'
 
+const StyledButton = styled(Button)`
+  background-color: ${props => props.theme.global.colors['neutral-1']};
+  border-radius: 4px;
+  color: ${props => props.theme.global.colors['neutral-6']};
```

Comment:
[minor] Just a minor note here that StyledButton will pretty much crash the page if the component is ever rendered without a Grommet `theme`, or a `theme` with incomplete values. ("can't find value color of undefined", etc)

In theory, replacing all those `.` with `?.` would make the code more robust. In practice, I can understand if `props?.theme?.global?.colors?.['neutral-6']` is a bit too messy to read, and it's a fair expectation for every FEM component to be rendered with a proper Grommet `theme`.